plugins/scaffolder-react/api-report.md

```diff
@@ -71,27 +98,38 @@ export type FieldExtensionComponent<_TReturnValue, _TInputProps> = () => null;
 // @public
 export interface FieldExtensionComponentProps<
   TFieldReturnValue,
-  TUiOptions = unknown,
-> extends FieldProps<TFieldReturnValue> {
+  TUiOptions = {},
+> extends PropsWithChildren<ScaffolderRJSFFieldProps<TFieldReturnValue>> {
   // (undocumented)
-  uiSchema: FieldProps['uiSchema'] & {
-    'ui:options'?: TUiOptions;
-  };
+  uiSchema: FieldExtensionUiSchema<TFieldReturnValue, TUiOptions>;
 }
 
 // @public
 export type FieldExtensionOptions<
   TFieldReturnValue = unknown,
-  TInputProps = unknown,
+  TUiOptions = unknown,
 > = {
   name: string;
   component: (
-    props: FieldExtensionComponentProps<TFieldReturnValue, TInputProps>,
+    props: FieldExtensionComponentProps<TFieldReturnValue, TUiOptions>,
   ) => JSX.Element | null;
-  validation?: CustomFieldValidator<TFieldReturnValue>;
+  validation?: CustomFieldValidator<TFieldReturnValue, TUiOptions>;
   schema?: CustomFieldExtensionSchema;
 };
 
+// @public
+export interface FieldExtensionUiSchema<TFieldReturnValue, TUiOptions>
+  extends UiSchema<TFieldReturnValue> {
+  // (undocumented)
+  'ui:options'?: TUiOptions & UIOptionsType<TFieldReturnValue>;
+}
+
+// @public
+export type FormProps = Pick<
+  FormProps_2,
+  'transformErrors' | 'noHtml5Validate'
+>;
```

Comment:
This seems strange, just those two fields in an exported type? Should it be moved over to the plugin instead? perhaps with a more specific name too